pkg/uefi/uefi.go

```diff
@@ -14,6 +14,19 @@ import (
 	"reflect"
 )
 
+var (
+	// ReadOnly breaks firmware modification operations, but optimizes
+	// memory and CPU consumption for read-only operations.
+	//
+	// DO NOT USE THIS OPTION UNLESS YOU KNOW WHAT ARE YOU DOING.
+	ReadOnly = false
```

Comment:
why not also add a flag to the utk command to set these two? This makes it easier to use from the cmdline

Reply:
I'm in general opposed to options with unknown bad consequences. 
DO NOT USE THIS OPTION UNLESS YOU KNOW WHAT ARE YOU DOING

Given that the tool runs in, basically, one second, this strikes me as optimization with high risk and low reward.

Further, statistically, you can't have a sample of one, so the measurement strikes me as too limited.

Reply:
> Further, statistically, you can't have a sample of one, so the measurement strikes me as too limited.

I'll perform this test on a set of firmwares and will share the results.